packages/atomic/stencil.config.ts

```diff
@@ -41,7 +41,7 @@ export const config: Config = {
     transformIgnorePatterns: [],
   },
   devServer: {
-    reloadStrategy: 'hmr',
+    reloadStrategy: 'pageReload',
```

Comment:
hmr in dev mode work really bad. It refreshes the page sometimes & some other times it just re-renders badly.
Setting it to page reload make the experience consistent and less prone to errors.